### ISSUE TITLE
Custom menu not supported

### ISSUE THREAD

**Issue by Lovor01** [polylang/polylang#900]
## Prerequisites

- [x] This is not a usage question (Those should be directed to the [community supported forum](https://wordpress.org/support/plugin/polylang), unless this is a question about Polylang Pro in which case you should [use the helpdesk](https://polylang.pro/support/)).
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest `master` branch of Polylang and the latest WordPress version.
- [x] This bug happens with only Polylang plugin active
- [x] This bug happens with a default WordPress theme active
- [x] I can reproduce this bug consistently

## Steps to reproduce the issue

1. In the theme, there is custom menu output, with changed id and classes:
```		<?php
			wp_nav_menu(
				array(
					'theme_location' => 'menu-1',
					'menu_id'        => 'primary-menu',
					'container'		  => 'nav',
					'container_id'	  => 'container-slide',
				)
			);
		?>
```
2. Upon activating Polylang, theme layout breaks, because css relies on custom container and container id
3. This happens because polylang outputs default menu wp_nav_menu();

## Expected behavior and actual behavior
Expected behavior: Respect menu customizations in php code.

**Comment by Chouby** [comment]
Sorry, I don't understand the issue. Could you please be more precise?
You mention you are using a default theme. Which one?
Starting from a fresh WP install with Polylang, how can I reproduce your issue on my test install?

**Comment by Lovor01** [comment]
I am not using default theme, I am using my custom theme. wp_nav_menu function call is not used with default parameters, but with parameters given above. Menu id and container id and tag are changed from their defaults (e.g. container is output with `nav` tag). Therefore, menu as given in my theme is not standard default menu, it is modified, but in Wordpress standard way. However, when Polylang plugin is activated, it does not respect given changes, but instead switches to default menu HTML output (as if wp_nav_menu is used with default parameters and not the ones I gave).

Polylang should respect custom wp_nav_menu call and output the same menu, as it was before activating it.

I hope I clarified the issue.

**Comment by Chouby** [comment]
>I am not using default theme

You checked the option.
>I hope I clarified the issue.

No. And I still don't reproduce. 

Here are the detailed steps of how I proceeded:
1. Start from Twenty Twenty one and replace the whole file by your code, except for the theme location where I used  one available in the theme, to avoid rewriting the registration too.
```php
wp_nav_menu(
	array(
		'theme_location'  => 'primary',
		'menu_id' => 'primary-menu',
		'container' => 'nav',
		'container_id' => 'container-slide',
	)
);
```
2. Create a menu with a single custom link menu item in backend assign it to the primary location.
Without Polylang, this outputs:
```html
<nav id="container-slide" class="menu-menu-container">
	<ul id="primary-menu" class="menu">
		<li id="menu-item-676" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-676">
			<a href="http://mysite.com">Home</a>
		</li>
	</ul>
</nav>
```
3. I then activated Polylang and assigned the same menu to the primary location in all languages.
4. I navigated through all languages and the html output is exactly the same as without Polylang.

I invite you to make the test as described above which demonstrates that Polylang doesn't modify the html output of the nav menu.
